### ISSUE TITLE
PLE1507 false positives

### ISSUE THREAD

**Issue by davidnemec** [astral-sh/ruff#3512]
Hi :wave:
with latest version that adds PLE1507 I'm getting errors with a following usage. But I would say they are false positives.

```python
import os

def dummy(env_prefix: str = ''):
    os.getenv(f'{env_prefix}NAME', '') # fail
    os.getenv(env_prefix + 'NAME', '') # fail
```

**Comment by charliermarsh** [comment]
Yeah definitely.
